hawkbit-repository/hawkbit-repository-jpa/pom.xml

```diff
@@ -91,6 +91,24 @@
    </dependencies>
 
    <build>
+      <resources>
+         <!-- We need to add also the default-resources, as this configuration overwrites the defaults -->
+         <resource>
+            <directory>src/main/java</directory>
```

Comment:
This default-resources seems to be included even if it is removed from this list. Please check if it is needed here.